From 8736ac20a1a7def9a024aa9c934b7b95ffdef373 Mon Sep 17 00:00:00 2001
From: Josh Blum <josh@joshknows.com>
Date: Tue, 17 Jan 2012 15:42:25 -0800
Subject: [PATCH 09/47] cmake: fix for cmake 2.6 with library extras mode

OUTPUT_NAME is a target output name that should work with both 2.6 and 2.8

LIBRARY_OUTPUT_NAME is new to 2.8 and more specific
---
 cmake/Modules/GrMiscUtils.cmake |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/cmake/Modules/GrMiscUtils.cmake b/cmake/Modules/GrMiscUtils.cmake
index 0e1f400..27003cf 100644
--- a/cmake/Modules/GrMiscUtils.cmake
+++ b/cmake/Modules/GrMiscUtils.cmake
@@ -155,7 +155,7 @@ function(GR_LIBRARY_FOO target)
         GR_LIBTOOL(TARGET ${target} DESTINATION ${GR_LIBRARY_DIR})
 
         #give the library a special name with ultra-zero soversion
-        set_target_properties(${target} PROPERTIES LIBRARY_OUTPUT_NAME ${target}-${LIBVER} SOVERSION "0.0.0")
+        set_target_properties(${target} PROPERTIES OUTPUT_NAME ${target}-${LIBVER} SOVERSION "0.0.0")
         set(target_name lib${target}-${LIBVER}.so.0.0.0)
 
         #custom command to generate symlinks
-- 
1.7.9.1

